crates/ide_assists/src/handlers/fix_visibility.rs

```diff
@@ -46,7 +46,17 @@ fn add_vis_to_referenced_module_def(acc: &mut Assists, ctx: &AssistContext) -> O
     let current_module = ctx.sema.scope(path.syntax()).module()?;
     let target_module = def.module(ctx.db())?;
 
-    let vis = target_module.visibility_of(ctx.db(), &def)?;
+    let vis = match def {
```

Comment:
should we not have `impl HasVisibility for ModuleDef`?

Reply:
I was debating that! We can't have it trivially, because it's unclear what would be the visibility of a `BuiltIn`. Hm, i think it actually can be public?

Reply:
Ooh ok yeah. Or maybe a method on `ModuleDef` that returns `Option<Visibility>`?

Reply:
But yeah public makes kind of sense